force-app/main/default/classes/LeadTriggerHandler.cls

```diff
@@ -13,6 +13,6 @@ public class LeadTriggerHandler extends TriggerHandler {
     }
 
     public override void beforeUpdate() {
-      
+      LeadService.UpdTriggerLeadStatus(newLeadList);
```

Comment:
don't forget to change name here as well

Reply:
I did .